pkg/services/sso/redhatsso_service.go

```diff
@@ -0,0 +1,136 @@
+package sso
+
+import (
+	"context"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/client/keycloak"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/client/redhatsso"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/services"
+	"github.com/golang/glog"
+	serviceaccountsclient "github.com/redhat-developer/app-services-sdk-go/serviceaccounts/apiv1internal/client"
+	"time"
+)
+
+type redhatssoService struct {
+	client redhatsso.SSOClient
+}
+
+func (r *redhatssoService) RegisterKafkaClientInSSO(accessToken string, kafkaNamespace string, orgId string) (string, *errors.ServiceError) {
+	// TODO: implement
+	return "", nil
+}
+
+func (r *redhatssoService) RegisterOSDClusterClientInSSO(accessToken string, clusterId string, clusterOathCallbackURI string) (string, *errors.ServiceError) {
+	// TODO: implement
+	return "", nil
+}
```

Comment:
@akoserwal It is not clear to me how we are going to implement this method with the new api

Reply:
Leave //todo